lib/rules/font-weight-notation/index.js

```diff
@@ -103,6 +103,16 @@ const rule = function(expectation, options) {
 			const weightValueOffset = decl.value.indexOf(weightValue);
 
 			if (expectation === 'numeric') {
+				if (decl.parent.type === 'atrule' && decl.parent.name.toLowerCase() === 'font-face') {
+					const weightValueNumbers = postcss.list.space(weightValue)
+
+					if (!weightValueNumbers.every(isNumbery)) {
```

Comment:
Could you add `reject` cases, please? Currently this condition is not tested.

Reply:
Done!